astropy/coordinates/tests/test_methods.py

```diff
@@ -0,0 +1,49 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+# TEST_UNICODE_LITERALS
+import itertools
+import numpy as np
+
+from ... import units as u
+from .. import SkyCoord, Longitude, Latitude
+from ...tests.helper import pytest
+
+
+class TestManipulation():
```

Comment:
Might it make sense to add a test somewhere (maybe in this file, maybe elsewhere) for higher-dimensional coordinates of some sort?  Like a transformation of a 3x4x2 ICRS coordinate to FK5 with some 3x4x2 array of equinoxes?